docs/widgetbook-cloud/quick-start.mdx

```diff
@@ -25,10 +25,11 @@ This quick start guide will help you set up and use Widgetbook Cloud efficiently
 2. Install and configure the Widgetbook CLI.
```

Comment:
Step 2, Point 2: _View details_ is outdated. It's now the _Settings_ button

Reply:
Also noticed that the _View details_ button is used on other section too, like the `Projects` page. Could you please search for it in all sections and change it to _Settings_? Thanks! :)

Reply:
fixed